examples/05-Q3D/Q3D_DC_IR.py

```diff
@@ -11,7 +11,7 @@
 
 import os
 import pyaedt
-
+from pyedb import Edb
```

Comment:
```suggestion
From PyEDB import EDB
```

Reply:
@PipKat sorry but this code will not wor. Python requires "from" and the exact name of the package